src/main/java/com/zufar/icedlatte/product/endpoint/ProductsEndpoint.java

```diff
@@ -67,4 +70,10 @@ public ResponseEntity<List<ProductInfoDto>> getProductsByIds(@RequestBody final
         return ResponseEntity.ok()
                 .body(products);
     }
+
```

Comment:
add open api description and make this method "Override"

Reply:
Just in case: `src/main/resources/api-specs/product-openapi.yaml`